g3doc/quick_reference.md

```diff
@@ -1928,16 +1928,16 @@ their operands into independently processed 128-bit *blocks*.
 
 All other ops in this section are only available if `HWY_TARGET != HWY_SCALAR`:
 
-*   `V`: `{u,i}` \
+*   `V`, `VI`: `{u,i}` \
     <code>VI **TableLookupBytes**(V bytes, VI indices)</code>: returns
     `bytes[indices[i]]`. Uses byte lanes regardless of the actual vector types.
     Results are implementation-defined if `indices[i] < 0` or `indices[i] >=
     HWY_MIN(Lanes(DFromV<V>()), 16)`. `VI` are integers, possibly of a different
```

Comment:
How about something like "`VI` are normal vectors with integer lane type possibly differing from the lanes of `V`. Note that TableLookup*Lanes* has an unspecified, potentially non-vector type type for the indices."?